### PR TITLE
added dummy script tag to prevent FOUC

### LINKED ISSUE #33793: FOUC in Firefox on login UI
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

login/ui

### Describe the bug

There is a flash of unstyled content on the login page. Even the password suggestion Popup placement is off. Because the autofocus sets the focus to the unstyled username input. And the position is not updated when the actual css kicks in.
Firefox also gives a warning: Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content.

I could not find out why exactly that happens. But, beside other things, it seems to be related to the module type of the scripts. <script type="module"... That somehow seems to affect the loading order of things.

### Version

26.0.0

### Regression

- [ ] The issue is a regression

### Expected behavior

There should be no flashes when loading the login page.

### Actual behavior

FOUC in Firefox on login UI

### How to Reproduce?

Just load the login page in Firefox.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #33793
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

### CODE REVIEW

File: themes/src/main/resources/theme/keycloak.v2/login/template.ftl [hunk 1]
```diff
@@ -92,10 +92,13 @@
             "${url.ssoLoginInOtherTabsUrl?no_esc}"
         );
     </script>
+    <script>
+      // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1404468
+      const isFirefox;
```
Comment (jonkoops) [inline themes/src/main/resources/theme/keycloak.v2/login/template.ftl:97]: How does this fix the issue exactly? There are plenty of script being loaded above already.

Comment (edewit) [reply themes/src/main/resources/theme/keycloak.v2/login/template.ftl:97]: I don't know how, I haven't looked at how firefox works, all I can say is that this removes the initial un-styled view

Comment (jonkoops) [reply themes/src/main/resources/theme/keycloak.v2/login/template.ftl:97]: I guess it just adds a blocking script to the page, the contents contents could be anything. What is the root cause of this issue? If it is a Firefox bug I would not bother fixing it with a hack, that is the responsibility of the Firefox team.

**Review by ssilvert — APPROVED** [review]
I'm OK with this.

**Pull request merged**